ando/tools/generator/utils.py

```diff
@@ -15,7 +15,7 @@ def save_tsv(dataframe, path_to_save):
         path to save the TSV file
 
     """
-    if Path(path_to_save).exists():
+    if Path(path_to_save).exists() and os.path.getsize(path_to_save) > 1:
```

Comment:
maybe adding a comment here would be good. I already forgot why we need the file size threshold...

Reply:
done